mysql/README.md

```diff
@@ -1,59 +1,131 @@
 # Overview
 
-Connect MySQL to Datadog in order to:
+The Datadog agent's MySQL check sends many database metrics to Datadog, including metrics for:
 
-* Visualize your database performance
-* Correlate the performance of MySQL with the rest of your applications
+* Query throughput
+* Query performance (average query run time, slow queries, etc)
+* Connections (currently open connections, aborted connections, errors, etc)
+* InnoDB (buffer pool metrics, etc)
 
-# Installation
+And [many more](https://github.com/DataDog/integrations-core/blob/master/mysql/metadata.csv). You can also invent your own metrics using custom queries.
```

Comment:
While it makes sense to link to the `metadata.csv` in the readme, if we're scraping these to build the docs, then we'll probably still have something to generate a better table out out of the metadata. It might be jumping the gun, but we should probably think about some sort of standard for linking to metadata that could easily work in readmes and also easily get regex'ed when the content is scraped.